my_robot:
    # Publish all joint states 
    # -----------------------------------
    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 50

    # Position Controllers 
    # ---------------------------------------
    left_front_wheel_velocity_controller:
        type: velocity_controllers/JointVelocityController
        joint: left_front_wheel_joint
        pid: {p: 3.0, i: 1.0, d: 0.0}
        
    left_rear_wheel_velocity_controller:
        type: velocity_controllers/JointVelocityController
        joint: left_rear_wheel_joint
        pid: {p: 3.0, i: 1.0, d: 0.0}

    right_front_wheel_velocity_controller:
        type: velocity_controllers/JointVelocityController
        joint: right_front_wheel_joint
        pid: {p: 3.0, i: 1.0, d: 0.0}

    right_rear_wheel_velocity_controller:
        type: velocity_controllers/JointVelocityController
        joint: right_rear_wheel_joint
        pid: {p: 3.0, i: 1.0, d: 0.0}

